web/styles/popovers.css

```diff
@@ -610,8 +610,6 @@ ul {
         border-radius: 0;
         border: none;
 
-        pointer-events: none;
-
```

Comment:
Interesting, I didn't notice that using this would cause the events to fall through the popover background. We don't need `pointer-events: all` below too in that case right?

Reply:
It might also be worth using `git log -S` to figure out the backstory for what commit added this, in case there's other related changes.

Reply:
f651c7fb18b0f783dee72807d28b46eaf8a75490 is the commit where this code was first introduced, I can't think of any purpose of this code.
```css
.message-info-popover,
    .user-info-popover {
        display: flex !important;
        justify-content: center;
        align-items: center;

        /* these are to override JS embedded inline styles. */
        top: 0 !important;
        left: 0 !important;
        margin: 0 !important;
        width: 100%;
        height: 100%;

        background-color: hsl(0deg 0% 0% / 70%);
        border-radius: 0;
        border: none;

        pointer-events: none;

        .popover-inner {
            background-color: hsl(0deg 0% 100%);
            pointer-events: all;
        }
    }
```
    
It seems like we were ensuring that nothing outside popover should be clickable, as `pointer-events: all;` is ensuring everything inside should work. 
22736084778d1d00a382681c178b634e19cc05aa, a4f5b0c635fc286f9fb95e0ab5b679bfecbf95fd are commits tackling bugs introduced because of `pointer-events` property.

Reply:
Ok. Also looking at [MDN doc](https://developer.mozilla.org/en-US/docs/Web/CSS/pointer-events) of this, we should be using `pointer-events: all` at all, since the support for HTML is experimental in this case. Can you do a cleanup of that as well?

Reply:
Sure, I will clean up all the `pointer-events: all;`, and manually test the effects.